src/mscorlib/shared/System/Reflection/MemberInfoSerializationHolder.cs

```diff
@@ -9,7 +9,6 @@
 
 namespace System.Reflection
 {
-    [Serializable]
 #if CORECLR
```

Comment:
If you get rid of this file, you can get rid of

  src\System\Reflection\MemberSerializationStringGenerator.cs

too. There are versions of this file in both CoreCLR and CoreRT.